app/uk/gov/hmrc/taxenrolmentassignmentfrontend/controllers/AccountCheckController.scala

```diff
@@ -95,7 +95,8 @@ class AccountCheckController @Inject() (
       case SA_ASSIGNED_TO_OTHER_USER if request.userDetails.hasPTEnrolment =>
         Future.successful(Redirect(routes.EnrolledPTWithSAOnOtherAccountController.view))
       case SA_ASSIGNED_TO_OTHER_USER   => Future.successful(Redirect(routes.SABlueInterruptController.view))
-      case SINGLE_OR_MULTIPLE_ACCOUNTS => Future.successful(Redirect(routes.EnrolledForPTController.view))
+      case SINGLE_ACCOUNT              => Future.successful(Redirect(routes.EnrolledForPTController.view))
```

Comment:
Is this the final redirect? In this case it should go directly to the service. Only multiple accounts would see the interstitial page.

Reply:
Fixed, single_account removed from the list